pkg/columns/columninfo_test.go

```diff
@@ -461,3 +461,50 @@ func TestColumnMatcher(t *testing.T) {
 		t.Errorf("Didn't expect mainString to have tag 'test'")
 	}
 }
+
+func TestColumnTemplates(t *testing.T) {
+	RegisterTypedTemplate[int8]("", "width:789")
+	RegisterTemplate("numbers", "width:123")
+	RegisterTypedTemplate[int32]("numbers", "width:456")
+
+	type testSuccess1 struct {
+		Int8  int8  `column:",template"`
+		Int16 int16 `column:",template:numbers"`
+		Int32 int32 `column:",template:numbers"`
```

Comment:
Is the `,` needed?

Reply:
Yes, that means "empty name", so it will get the name from the field itself (`Int8`, `Int16` and `Int32`) - it's basically the same way the JSON tags work (e.g. `json:",omitempty"`).